src/AppBundle/Controller/DefaultController.php

```diff
@@ -31,4 +31,30 @@ public function contactAction()
         return $this->render('default/contact.html.twig');
 
     }
+    /**
+     * @Route("/catalogue", name="catalogue")
+     */
+    public function catalogueAction()
+    {
+        $em = $this->getDoctrine()->getManager(); //get the database manager
+
+        $plantsSpecification = $em->getRepository('PlantBundle:PlantSpecification')->findAll(); // get the profiles
+
+        return $this->render('default/catalogue.html.twig', array('plants' => $plantsSpecification));
+    }
+    /**
+     * @Route("/catalogue/{$id}", name="showPlant")
```

Comment:
`@Route("/catalogue/{id}", name="showPlant")`
will make the work. Dollar `$id` cannot be used in annotations